pmm-app/src/pmm-dbaas/components/XtraDB/XtraDB.service.ts

```diff
@@ -7,7 +7,17 @@ export const XtraDBService = {
     return apiRequestManagement.post<any, any>('/DBaaS/XtraDBClusters/List', kubernetes);
   },
   addXtraDBCluster(xtradbCluster: XtraDBCluster) {
-    return apiRequestManagement.post<XtraDBClusterAPI, any>('/DBaaS/XtraDBCluster/Create', toAPI(xtradbCluster));
+    return apiRequestManagement.post<XtraDBClusterAPI, any>(
+      '/DBaaS/XtraDBCluster/Create',
+      toAPI(xtradbCluster),
+    );
+  },
+  deleteXtraDBClusters(xtradbCluster) {
+    const toAPI = (cluster) => ({
+      kubernetes_cluster_name: cluster.clusterName,
+    });
+
+    return apiRequestManagement.post<any, any>('/DBaaS/XtraDBClusters/Delete', toAPI(xtradbCluster));
```

Comment:
The request is wrong, it should be `/DBaaS/XtraDBCluster/Delete`. And the actual payload format should have the `name` of the cluster as well.